♻️ Chore: Update getting started guide to include changes needed for package.json start scripts (#267)
astrolicious/studiocms: www/docs/src/content/docs/start-here/getting-started.mdx

});
```

> ## Configure your `package.json`

Reviewer: ```suggestion
## Configure your `package.json` scripts
```

> Setup your package.json scripts to include the `--remote` flag for build, and optionally for dev. (You can also run the dev command as shown in the "Running your StudioCMS Project" section)

Reviewer: ```suggestion
Setup your `package.json` scripts to include the `--remote` flag for build, and optionally for dev. (You can also run the dev command as shown in the ["Running your StudioCMS Project"](#running-your-studiocms-project) section)
```

```json title="package.json"
{
    "name": "my-studiocms-project",
    "scripts": {
        "dev": "astro dev --remote",
        "build": "astro check & astro build --remote",
        "astro": "astro"
    }
}
```

:::note
Astro StudioCMS requires SSR mode to work correctly. Make sure to set `output: 'server'` and an adapter that supports SSR in your `astro.config.mjs` file.

> :::

Reviewer: This should be under the previous header as it relates to the Astro config, not the `package.json` file

Reviewer 2: whoops... working from my windows install at the moment so I'm on the web browser and didn't notice that got put in the wrong place... 😅 fixed in [951dce1](https://github.com/astrolicious/studiocms/pull/267/commits/951dce1f83a75eb330c02552d386388ea47f67f7)

Astro StudioCMS requires SSR mode to work correctly. Make sure to set `output: 'server'` and an adapter that supports SSR in your `astro.config.mjs` file.
:::

## Configure your `package.json` scripts

Setup your `package.json` scripts to include the `--remote` flag for build, and optionally for dev. (You can also run the dev command as shown in the ["Running your StudioCMS Project"](#running-your-studiocms-project) section)

```json title="package.json"
{
    "name": "my-studiocms-project",
    "scripts": {
        "dev": "astro dev --remote",
        "build": "astro check & astro build --remote",
        "astro": "astro"
    }
}
```

## Running your StudioCMS project

> Thanks to the power of Astro running StudioCMS is as easy as running the dev command for local preview, or building and deploying to your server, for the basics of how to use it locally without building here is what you need to do.
> 

Reviewer: This doesn't provide any useful info, so I'd probably remove it unless there's something specific to call out here.
```suggestion
```

Reviewer 2: I admit it's mainly to help with spacing of the headings at the moment... 😅

I was thinking once [PR#11385](https://github.com/withastro/astro/pull/11385) switch this small section to mention specifically the AstroDB environment variable options for running with a Self-hosted LibSQL vs Astro Studio

Reviewer: h2 -> h3 is fine, and if we want to add something later, that's fine. still, no reason to block this whether it's removed/replaced/remains, so I'll approve

### First time Setup (or during updates if the tables schema is updated)

To start your Astro project, run the following commands in your terminal:

    <PackageManagers type="exec" args="astro db link" />
    <PackageManagers type="exec" args="astro db push" />
    <PackageManagers type="run" args="dev --remote" />

> After running the commands, you should see a message indicating that your project is running at `localhost:4321`. During first time setup of StudioCMS you will also see a message indicating that you need to setup your StudioCMS install at `http://localhost:4321/start`

Reviewer: ```suggestion
After running the commands, you should see a message indicating that your project is running at `localhost:4321`. When first setting up StudioCMS, you will prompted to finish configuring StudioCMS at http://localhost:4321/start
```
